src/workers/newNotificationMail.ts

```diff
@@ -61,6 +61,7 @@ const notificationToTemplateId: Record<NotificationType, string> = {
   post_mention: '',
   promoted_to_moderator: 'd-b1dbd1e86ee14bf094f7616f7469fee8',
   squad_subscribe_to_notification: '',
+  collection_updated: '',
```

Comment:
Checking with product if email is required here
https://dailydotdev.slack.com/archives/C0601MRBJJU/p1701333105563279

Reply:
We can do email in separate PR to not block this one. I'll log it.

Reply:
So nothing from my side to change here?